src/routes/safe/components/Balances/SendModal/index.tsx

```diff
@@ -134,21 +135,28 @@ const SendModal = ({
             initialValues={tx as ReviewTxProp}
             onClose={onClose}
             onReview={handleTxCreation}
-            recipientAddress={recipientAddress}
+            recipientAddress={recipient}
             selectedToken={selectedToken as string}
             amount={tokenAmount}
           />
         )}
 
         {activeScreen === 'sendFundsReviewTx' && (
-          <ReviewSendFundsTx onClose={onClose} onPrev={() => setActiveScreen('sendFunds')} tx={tx as ReviewTxProp} />
+          <ReviewSendFundsTx
+            onClose={onClose}
+            onPrev={() => {
+              setRecipient((tx as ReviewTxProp).recipientAddress)
```

Comment:
Why is tx `unknown` and not `ReviewTxProp` in the first place?

Reply:
If it changes types, how do we know it's ReviewTxProp in this case? It seems at some point it does get a definitive type. I'd rather make it explicit instead of writing `as ReviewTxProp` in several places.